api/tools/src/main/java/org/pmiops/workbench/tools/BulkCloneWorkspacesTool.java

```diff
@@ -258,6 +263,7 @@ public CommandLineRunner run(
               workspaceService.getWorkspace(
                   dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getFirecloudName());
           CloneWorkspaceRequest request = createCloneRequest(apiWorkspace);
+          request.getWorkspace().setCdrVersionId(String.valueOf(defaultCdr.getCdrVersionId()));
```

Comment:
nit: probably cleaner to pass this version ID to `createCloneRequest`, or just set it in there